app/bundles/CoreBundle/Assets/js/1a.content.js

```diff
@@ -1400,11 +1416,22 @@ Mautic.activateLiveSearch = function (el, searchStrVar, liveCacheVar) {
         }
 
         //searchStr in MauticVars[liveCacheVar] ||
-        if ((!searchStr && MauticVars[searchStrVar].length) || diff >= 3 || spaceKeyPressed || enterKeyPressed) {
+        if ((!searchStr && MauticVars[searchStrVar].length)
+            || diff >= 3
+            || spaceKeyPressed
+            || enterKeyPressed
+        ) {
             MauticVars[searchStrVar] = searchStr;
             event.data.livesearch = true;
 
-            Mautic.filterList(event,
+            // Reset active index for global search
+            if (mQuery(el).attr('id') === 'globalSearchInput') {
+                activeIndex = -1;
+                mQuery('#globalSearchResults .gsearch--results-item').removeClass('active');
```

Comment:
Since `id` of the element equals to the `globalSearchInput`:

```js
if (mQuery(el).attr('id') === 'globalSearchInput') {
```

the `el` object is an instance of the `#globalSearchResults` element, so we can shorten the selector (without the `#globalSearchResults`) by referencing the `el`.

```suggestion
                mQuery('.gsearch--results-item', el).removeClass('active');
```

question: Could you please check that?